rust/connlib/snownet/src/node.rs

```diff
@@ -144,6 +144,8 @@ where
         for allocation in self.allocations.values_mut() {
             allocation.refresh(now);
         }
+
+        self.bindings_and_allocations_drain_events();
```

Comment:
This is actually wrong. `refresh` is async so we don't have any events here. But we should add this function to the top of `poll_event` to ensure we drain them in time.

Reply:
Should I do it with this PR?

Reply:
If you want to. I just don't want to add this here because it is pointless :)

Reply:
added :)